test/jdk/java/nio/file/Files/CopyMoveVariations.java

```diff
@@ -72,7 +72,7 @@ enum PathType {
     static {
         Path tmp = null;
         try {
-            tmp = Files.createTempFile("this", "that");
+            tmp = Files.createTempFile(Path.of("."), "this", "that");
```

Comment:
Could you change `tmp` to be a constant, for example
```
    private static final Path CURRENT_DIR;
```
(or `CWD`) and change `current` to be `CURRENT_DIR`?

Reply:
> Could you change `tmp` to be a constant, for example
> 
> ```
>     private static final Path CURRENT_DIR;
> ```

```
74         try {
75             currentDir = Files.createTempFile(Path.of("."), "this", "that");
76             SUPPORTS_POSIX_PERMISSIONS =
77                 Files.getFileStore(currentDir).supportsFileAttributeView("posix");
78         } catch (IOException cause) {
79             throw new UncheckedIOException(cause);
80         } finally {
81             if (currentDir != null) {
```

After change tmp to be a constant currentDir,  variable currentDir can not be initialized in the finally block.

```
private static final Path currentDir;
```